electron/app/chatBro/db/index.ts

```diff
@@ -0,0 +1,13 @@
+import * as _ from 'lodash';
+import * as sqlite3 from 'sqlite3';
+
+export function initializeDB(): sqlite3.Database {
+  const sqldb = sqlite3.verbose();
+  const db = new sqldb.Database(`${process.env.HOME}/Desktop/chat.db`);
```

Comment:
`${process.env.HOME}/Desktop/chat.db` should read from the constants file I have set up ... going to add a commit